packages/react-ai/src/components/AIConversation/createProvider.tsx

```diff
@@ -55,29 +61,33 @@ export default function createProvider({
     return (
       <ElementsProvider elements={elements}>
         <ControlsProvider controls={controls}>
-          <SuggestedPromptProvider suggestedPrompts={suggestedPrompts}>
-            <ResponseComponentsProvider responseComponents={responseComponents}>
-              <ConversationDisplayTextProvider {..._displayText}>
-                <ConversationInputContextProvider>
-                  <SendMessageContextProvider
-                    handleSendMessage={handleSendMessage}
-                  >
-                    <AvatarsProvider avatars={avatars}>
-                      <ActionsProvider actions={actions}>
-                        <MessageVariantProvider variant={variant}>
-                          <MessagesProvider messages={messages}>
-                            <LoadingContextProvider isLoading={isLoading}>
-                              {children}
-                            </LoadingContextProvider>
-                          </MessagesProvider>
-                        </MessageVariantProvider>
-                      </ActionsProvider>
-                    </AvatarsProvider>
-                  </SendMessageContextProvider>
-                </ConversationInputContextProvider>
-              </ConversationDisplayTextProvider>
-            </ResponseComponentsProvider>
-          </SuggestedPromptProvider>
+          <AttachmentProvider allowAttachments={allowAttachments}>
```

Comment:
Nit: Should maybe be below the more `static` providers like `SuggestedPrompts` and `ResponseComponents` since those are unlikely to change with `createAIConversation`